### ISSUE TITLE
"Unicode property not found" breaking build

### ISSUE THREAD

**Issue by Gip-Gip** [rp-rs/pio-rs#40]
Cleared my Cargo.lock and tried to build but I ended up being met with this error:

```
error: failed to run custom build command for `pio-parser v0.2.0`

Caused by:
  process didn't exit successfully: `/home/thompsonc/Projects/msbaker-firmware/target/release/build/pio-parser-67e78b185c5e4962/build-script-build` (exit status: 1)
  --- stdout
  processing file `/home/thompsonc/.cargo/registry/src/github.com-1ecc6299db9ec823/pio-parser-0.2.0/src/pio.lalrpop`
  /home/thompsonc/.cargo/registry/src/github.com-1ecc6299db9ec823/pio-parser-0.2.0/src/pio.lalrpop:273:10: 273:44 error: invalid regular expression: regex parse error:
      (\p{ID_Start}|_)\p{ID_Continue}*
       ^^^^^^^^^^^^
  error: Unicode property not found

  --- stderr
    Symbol = r"(\p{ID_Start}|_)\p{ID_Continue}*";
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I don't know why this is just now an issue but I can no longer compile which kinda sucks. Perhaps a breaking issue way upstream was pushed but not sure.

Any help is greatly appreciated.

**Comment by jannic** [comment]
This is caused by https://github.com/lalrpop/lalrpop/issues/750
Adding the dependency `regex-syntax = { version = "0.6", default_features = false, features = ["unicode"] }` to your `Cargo.toml` should work around the issue.
@ithinuel is currently working on building that workaround into pio-rs, but you can also add it to your `msbaker-firmware`.

**Comment by Gip-Gip** [comment]
Fixed in 0.2.2, many thanks!
